training/GPController.py

```diff
@@ -0,0 +1,138 @@
+import matplotlib.pyplot as plt
+import gpytorch
+import torch
+import time
+import data.dataloader as dataloader
+from training.ExactGP import ExactGPModel
+from training.BatchIndependentMultiTaskGP import BatchIndependentMultiTaskGPModel
+from training.MultiTaskGP import MultitaskGPModel
+from torch.utils.data import DataLoader, TensorDataset
+import numpy as np
+
+torch.cuda.memory_summary(device=None, abbreviated=False)
```

Comment:
This line causes a crash on machines without CUDA:

```
$ python gp_rl_controller.py
Traceback (most recent call last):
  File "/home/casey/RL_Project/gp_rl_controller.py", line 1, in <module>
    from training.GPController import GPModel
  File "/home/casey/RL_Project/training/GPController.py", line 12, in <module>
    torch.cuda.memory_summary(device=None, abbreviated=False)
  File "/usr/lib/python3.11/site-packages/torch/cuda/memory.py", line 518, in memory_summary
    current = stats[prefix + "current"]
              ~~~~~^^^^^^^^^^^^^^^^^^^^
KeyError: 'allocated_bytes.all.current'
```

Reply:
Seems similar to the `nvidia-smi` shell command, which can be checked separately if desired, so I recommend just deleting this from here.